packages/server-api/src/plugin.ts

```diff
@@ -129,7 +129,7 @@ export interface Plugin {
    *
    * @category Configuration
    */
-  schema: object | (() => object)
+  schema?: object | (() => object)
```

Comment:
I don't think a plugin [works without a schema](https://github.com/SignalK/signalk-server/blob/master/src/interfaces/plugins.ts#L170-L191) so why optional?

Reply:
You are right. I don't remember why I changed this. Reverted.